source/includes/inkling2-reference/_curriculums.html.md

```diff
@@ -0,0 +1,122 @@
+# Curriculums
+
+```inkling2--code
+concept MyConcept(input): OutputType {
+  curriculum {
+    source MySimulator
+
+    # Lessons specified here
+
+  }
+}
+```
+
+The `curriculum` statement is used within a [concept][2] to specify how the concept should be trained.
+
+The [lessons][1] defined within the curriculum can be used to create a staged teaching plan.
+
+> Curriculum Syntax
+
+```inkling2--syntax
+curriculum {
+  source <simulatorReference>
+
+  [lessonStatement]*
+}
+```   
+
+### Usage
+
+There can be only one curriculum per concept, and every concept must have a curriculum.
+
+Every curriculum must provide a `source` clause that specifies the data source for teaching the concept. Currently, only [simulators][3] are supported as data sources, so the `source` keyword must be followed by a simulator declaration or the name of a simulator declared at the global scope of the program.
```

Comment:
can simulators really be declared inline? I suggest requiring them to always be at the global scope for now...